lib/utils/department_enum.dart

```diff
@@ -1,34 +1,25 @@
-enum DepartmentEnum {
-  aids,
-  cs,
-  it,
-  biomed,
-  biotech,
-  chem,
-  extc,
-  fe;
+enum DepartmentEnum { aids, cs, it, biomed, biotech, chem, extc, fe }
 
-  String get name {
-    switch (this) {
-      case DepartmentEnum.aids:
-        return "AI & DS";
-      case DepartmentEnum.cs:
-        return "Computer Engineering";
-      case DepartmentEnum.it:
-        return "Information Technology";
-      case DepartmentEnum.biomed:
-        return "Biomedical Engineering";
-      case DepartmentEnum.biotech:
-        return "Bio-Tech Engineering";
-      case DepartmentEnum.chem:
-        return "Chemical Engineering";
-      case DepartmentEnum.extc:
-        return "EXTC";
-      case DepartmentEnum.fe:
-        return "First Year";
-    }
-  }
```

Comment:
This is a better implementation. Don't follow stupid insta reels (I did this mistake).